adobe_vipm/adobe/constants.py

```diff
@@ -77,15 +79,18 @@
 REGEX_SANITIZE_COMPANY_NAME = re.compile(r"[^\w ,.＆&・\'()（）\\\"/-]")
 REGEX_SANITIZE_FIRST_LAST_NAME = re.compile(r"[^\p{L} 0-9,.＆&' \-\\\"]")
 
-# TODO: group make statuses in an enum - will be much clearer and easier to use
-STATUS_3YC_ACCEPTED = "ACCEPTED"
-STATUS_3YC_DECLINED = "DECLINED"
-STATUS_3YC_COMMITTED = "COMMITTED"
-STATUS_3YC_ACTIVE = "ACTIVE"
-STATUS_3YC_REQUESTED = "REQUESTED"
-STATUS_3YC_NONCOMPLIANT = "NONCOMPLIANT"
-STATUS_3YC_EXPIRED = "EXPIRED"
-OFFER_TYPE_LICENSE = "LICENSE"
-OFFER_TYPE_CONSUMABLES = "CONSUMABLES"
+
+class ThreeYearCommitmentStatus(Enum):
```

Comment:
This is great! thanks for that. I have one suggestion to use @unique. Also could you please add TEMPORARY group replacing line adobe_vipm/flows/sync.py:56. What I mean is something like that:

class ThreeYearCommitmentStatus(Flag):
    ACCEPTED = auto()
    DECLINED = auto()
    COMMITTED = auto()
    ACTIVE = auto()
    REQUESTED = auto()
    NONCOMPLIANT = auto()
    EXPIRED = auto()
    
    TEMPORARY = REQUESTED | ACCEPTED
    
    def __getattribute__(self, name):
        if name == 'value':
            return self.name
        
        return super().__getattribute__(name)

Reply:
Lukasz, I think that's overcomplicating the solution, since an enum represents a set of possible values. So, if you want to do shortcuts, just do it :-)

```
TEMPORARY_3YC_STATUSES = (ThreeYearCommitmentStatus.ACCEPTED, ThreeYearCommitmentStatus. REQUESTED)
```